2-WebApp-graph-user/2-3-Multi-Tenant/README.md

```diff
@@ -216,6 +216,10 @@ services.AddAuthentication(AzureADDefaults.AuthenticationScheme)
 
  You can read about the various endpoints of the Microsoft Identity Platform [here](https://docs.microsoft.com/azure/active-directory/develop/active-directory-v2-protocols#endpoints).
 
+#### Implications of signing-in guest users on /common endpoint
+
+Please note that if you sign-in guest users at the `/common` endpoint, they will be directed to their home tenant for signing-in. So, if your multi-tenant app cares about applying a tenant specific CA policies, group assignments or app roles to be applied to the guest users, the app should sign the guest user on the **tenanted endpoint** (https://login.microsoftonline.com/{tenantId}) instead of the `/common` endpoint.
```

Comment:
Should 'CA' be spelled out as 'Conditional Access'? I think generally an abbreviation should be spelled out the first time it's used in the text unless it's common.

Reply:
fixed